src/components/Label.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Label extends React.Component {
```

Comment:
Could you, please, replace it with stateless component?

Reply:
Done